mutation/atomic_cell.cc

```diff
@@ -79,23 +79,20 @@ compare_atomic_cell_for_merge(atomic_cell_view left, atomic_cell_view right) {
         return left.is_live() ? std::strong_ordering::less : std::strong_ordering::greater;
```

Comment:
I think that giving a live cell preference to not-live cell even if the not-live cell was written after the live cell is not obvious at all.

Reply:
We have no way to determine this. We use timestamps to determine order and we are in a tie. Our ordering broke down, and we just need *some* order (bonus if it is also intuitive).

Reply:
@denesb Don't we always store a coordinator time *together* with a "timestamp" (which can be overridden with `USING TIMESTAMP`)? 
Or we only have it if there if data is TTLed and we have an `expire_at`?

Reply:
@bhalevy is the code with this patch copies the latest C* logic 1-to-1?

Reply:
@vladzcloudius it doesn't _copy_ the C* logic but it is compatible with it, using a different implementation.
Also, it extend C*'s algorithm as we examine the cell remaining ttl, while C* doesn't so that.  They stop processing at the expiry time and if both cells expire at the same time, their value is compared.